arch/arm/soc/nordic_nrf5/nrf52/Kconfig.soc

```diff
@@ -26,3 +26,11 @@ config SOC_NRF52840_QIAA
 	select SOC_NRF52840
 
 endchoice
+
+config NRF52_ARM_MPU_ENABLE
```

Comment:
The STM32 implementation uses STM32_ARM_MPU_ENABLE, so I also used that pattern in this patch.  However, the ARM Beetler uses "ARM_MPU_ENABLE" (w/o SoC at all) so there is no standard, I'm open to using your format.